`make checks` is failing on master branch

Fails with this error:

```
$ make checks
Running validation scripts...
RUN hack/check-schema-changes.sh
PASSED hack/check-schema-changes.sh in 1s
RUN hack/check-skaffold-builder.sh
PASSED hack/check-skaffold-builder.sh in 0s
RUN hack/check-samples.sh
PASSED hack/check-samples.sh in 4s
RUN hack/check-docs.sh
PASSED hack/check-docs.sh in 0s
RUN hack/test-generated-proto.sh
Sending build context to Docker daemon  149.6kB
Step 1/21 : FROM golang:1.13.4-alpine3.10 AS generate-files
 ---> 3024b4e742b0
Step 2/21 : RUN apk add --no-cache protobuf=3.6.1-r1 unzip jq moreutils
 ---> Running in 9e3764c85d11
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/community/x86_64/APKINDEX.tar.gz
(1/12) Installing oniguruma (6.9.4-r0)
(2/12) Installing jq (1.6-r0)
(3/12) Installing libbz2 (1.0.6-r7)
(4/12) Installing perl (5.28.3-r0)
(5/12) Installing perl-timedate (2.30-r1)
(6/12) Installing perl-io-tty (1.12-r5)
(7/12) Installing perl-ipc-run (20180523.0-r1)
(8/12) Installing moreutils (0.63-r1)
(9/12) Installing libgcc (8.3.0-r0)
(10/12) Installing libstdc++ (8.3.0-r0)
(11/12) Installing protobuf (3.6.1-r1)
(12/12) Installing unzip (6.0-r6)
Executing busybox-1.30.1-r2.trigger
OK: 53 MiB in 27 packages
Removing intermediate container 9e3764c85d11
 ---> 7e072bde7537
Step 3/21 : WORKDIR /protoc
 ---> Running in 873bce8b94e1
Removing intermediate container 873bce8b94e1
 ---> 9d4a0987b88f
Step 4/21 : ENV PROTOC_VERSION=3.11.0
 ---> Running in 02e75ee8008c
Removing intermediate container 02e75ee8008c
 ---> b06e234f8f8b
Step 5/21 : RUN wget -O protoc.zip https://github.com/google/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip
 ---> Running in 35de09889c90
Connecting to github.com (140.82.118.3:443)
Connecting to github.com (140.82.118.3:443)
Connecting to github-production-release-asset-2e65be.s3.amazonaws.com (52.216.184.115:443)
protoc.zip            19% |******                          |  305k  0:00:04 ETA
protoc.zip            47% |***************                 |  730k  0:00:02 ETA
protoc.zip            75% |************************        | 1172k  0:00:00 ETA
protoc.zip           100% |********************************| 1547k  0:00:00 ETA
Removing intermediate container 35de09889c90
 ---> ef7cd747c934
Step 6/21 : RUN unzip protoc.zip
 ---> Running in 92ffb05f0aa2
Archive:  protoc.zip
   creating: include/
   creating: include/google/
   creating: include/google/protobuf/
  inflating: include/google/protobuf/type.proto
  inflating: include/google/protobuf/duration.proto
  inflating: include/google/protobuf/empty.proto
  inflating: include/google/protobuf/wrappers.proto
  inflating: include/google/protobuf/field_mask.proto
   creating: include/google/protobuf/compiler/
  inflating: include/google/protobuf/compiler/plugin.proto
  inflating: include/google/protobuf/struct.proto
  inflating: include/google/protobuf/any.proto
  inflating: include/google/protobuf/descriptor.proto
  inflating: include/google/protobuf/source_context.proto
  inflating: include/google/protobuf/timestamp.proto
  inflating: include/google/protobuf/api.proto
   creating: bin/
  inflating: bin/protoc
  inflating: readme.txt
Removing intermediate container 92ffb05f0aa2
 ---> ddb065096e81
Step 7/21 : WORKDIR /grpc-gateway
 ---> Running in e9ef759e1413
Removing intermediate container e9ef759e1413
 ---> c83ca31100d4
Step 8/21 : RUN wget -q -O- https://github.com/grpc-ecosystem/grpc-gateway/tarball/v1.9.1 | tar --strip-components 1 -zx
 ---> Running in 34f384e6b87d
Removing intermediate container 34f384e6b87d
 ---> 082fc1b35e31
Step 9/21 : WORKDIR /tmp
 ---> Running in eb75f99dc8ae
Removing intermediate container eb75f99dc8ae
 ---> 824f4bfe0014
Step 10/21 : ENV GOPROXY=https://proxy.golang.org
 ---> Running in a63622aafd13
Removing intermediate container a63622aafd13
 ---> 9a61bfe8f416
Step 11/21 : ENV GO111MODULE=on
 ---> Running in 725c6435b63a
Removing intermediate container 725c6435b63a
 ---> 9c50515310cc
Step 12/21 : RUN go get     github.com/golang/protobuf/protoc-gen-go@v1.3.2     github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v1.9.1     github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@v1.9.1     github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc@v1.3.0
 ---> Running in 9685a878337a
go: finding github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc v1.3.0
go: finding github.com/pseudomuto/protoc-gen-doc/cmd v1.3.0
go: finding github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway v1.9.1
go: finding github.com/pseudomuto/protoc-gen-doc v1.3.0
go: finding github.com/grpc-ecosystem v1.9.1
go: finding github.com/grpc-ecosystem/grpc-gateway v1.9.1
go: finding github.com v1.3.0
go: finding github.com v1.9.1
go: finding github.com/golang/protobuf v1.3.2
go: finding github.com v1.3.2
go: finding github.com/pseudomuto v1.3.0
go: finding github.com/golang/protobuf/protoc-gen-go v1.3.2
go: finding github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger v1.9.1
go: finding github.com/golang v1.3.2
go: downloading github.com/golang/protobuf v1.3.2
go: downloading github.com/grpc-ecosystem/grpc-gateway v1.9.1
go: downloading github.com/pseudomuto/protoc-gen-doc v1.3.0
go: extracting github.com/golang/protobuf v1.3.2
go: extracting github.com/grpc-ecosystem/grpc-gateway v1.9.1
go: extracting github.com/pseudomuto/protoc-gen-doc v1.3.0
go: downloading github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
go: finding github.com/mwitkow/go-proto-validators v0.3.0
go: finding github.com/pseudomuto/protokit v0.2.0
go: finding github.com/Masterminds/sprig v2.22.0+incompatible
go: extracting github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
go: downloading github.com/ghodss/yaml v1.0.0
go: downloading github.com/mwitkow/go-proto-validators v0.3.0
go: downloading google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8
go: downloading github.com/pseudomuto/protokit v0.2.0
go: downloading github.com/Masterminds/sprig v2.22.0+incompatible
go: extracting github.com/ghodss/yaml v1.0.0
go: downloading gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
go: finding github.com/lyft/protoc-gen-validate v0.4.0
go: extracting github.com/mwitkow/go-proto-validators v0.3.0
go: downloading github.com/lyft/protoc-gen-validate v0.4.0
go: extracting github.com/Masterminds/sprig v2.22.0+incompatible
go: extracting github.com/pseudomuto/protokit v0.2.0
go: extracting gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
go: extracting github.com/lyft/protoc-gen-validate v0.4.0
go: extracting google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8
go: github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc imports
	github.com/pseudomuto/protoc-gen-doc/extensions/lyft_validate imports
	github.com/lyft/protoc-gen-validate/validate: github.com/lyft/protoc-gen-validate@v0.4.0: parsing go.mod:
	module declares its path as: github.com/envoyproxy/protoc-gen-validate
	        but was required as: github.com/lyft/protoc-gen-validate
The command '/bin/sh -c go get     github.com/golang/protobuf/protoc-gen-go@v1.3.2     github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v1.9.1     github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@v1.9.1     github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc@v1.3.0' returned a non-zero code: 1

Generated proto files aren't updated. Please run ./hack/generate-proto.sh
FAILED hack/test-generated-proto.sh
make: *** [checks] Error 1
```

--- a/hack/proto/Dockerfile
+++ b/hack/proto/Dockerfile
@@ -30,7 +30,7 @@ RUN go get \
     github.com/golang/protobuf/protoc-gen-go@v1.3.2 \
     github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v1.9.1 \
     github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@v1.9.1 \
-    github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc@v1.3.0
+    github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc@v1.3.2
 
 WORKDIR /proto
 COPY skaffold.proto markdown.tmpl ./

